src/services/directory-notification.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import ReconnectingWebSocket from 'reconnecting-websocket';
+import { getToken, getUrlWithToken } from '../utils/rest-api';
+
+const PREFIX_DIRECTORY_NOTIFICATION_WS = `${process.env.REACT_APP_WS_GATEWAY}/directory-notification`;
+
+export function connectDeletedStudyNotificationsWebsocket(studyUuid) {
+    // The websocket API doesn't allow relative urls
+    const wsbase = document.baseURI
+        .replace(/^http:\/\//, 'ws://')
+        .replace(/^https:\/\//, 'wss://');
```

Comment:
i think we can make it a function in utils

Reply:
rest-api is already in utils folder.
So I planned in next PRs for this TS to move all api calls functions to services folder and preserve only utils functions on rest-api.

thank you

Reply:
done, thank you